drivers/bridge/bridge.go

```diff
@@ -667,16 +707,16 @@ func (d *driver) createNetwork(config *networkConfiguration) (err error) {
 
 	// Add inter-network communication rules.
 	setupNetworkIsolationRules := func(config *networkConfiguration, i *bridgeInterface) error {
-		if err := network.isolateNetwork(networkList, true); err != nil {
-			if err = network.isolateNetwork(networkList, false); err != nil {
+		if err := network.isolateNetwork(iptables.IPv4, networkList, true); err != nil {
+			if err = network.isolateNetwork(iptables.IPv4, networkList, false); err != nil {
```

Comment:
same as above

Reply:
moved handling of IP version inside `network.isolateNetwork` and also handle IPv6